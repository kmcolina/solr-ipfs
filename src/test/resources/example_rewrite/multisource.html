<html>
<head>
    <title>Some page</title>
    <link rel="stylesheet" type="text/css" href="styles/index_o1.css" />
</head>
<h1>srcset testing</h1>

<p>An image with absolute srcset: <img src="http://example.com/foo_o2.jpg" srcset="http://example.com/fooF_o3.jpg 1x, http://example.com/fooF2_o4.jpg 2x"/></p>
<p>An image with mixed casing and spacing: <img src="http://example.com/foo_o2.jpg" srcSet="http://example.com/fooF_o3.jpg 720w , http://example.com/fooF2_o4.jpg 730W"/></p>
<p>An image with absolute srcset and post-space: <img src="http://example.com/foo_o2.jpg" srcset="http://example.com/fooF_o3.jpg 1x, http://example.com/fooF2_o4.jpg 2x "/></p>
<p>An image with absolute srcset and substring trickery: <img src="http://example.com/foo_o2.jpg" srcset="http://example.com/foo_o2.jpg 1x, http://example.com/foo_o2.jpg.bar_o5.jpg 2x"/></p>
<p>An image with relative srcset: <img src="fooB_o10.jpg" srcset="fooB_o10.jpg 1x, fooB2_o11.jpg 2x"/></p>
<p>An image with relative srcset and substring trickery: <img src="fooS_o21.jpg" srcset="fooS_o21.jpg 1x, barfooS_21.jpg_o22.jpg 2x"/></p>
<p>An image with root srcset: <img src="/fooC_o30.jpg" srcset="/fooC_o30.jpg 1x, /fooC2_o31.jpg 2x"/></p>
<p>Reproduction of specific problem: <img src="/fooC_o30.jpg" srcset="https://asset.dr.dk/imagescaler/?protocol=https&amp;server=www.dr.dk&amp;file=%2Fimages%2Fother%2F2021%2F01%2F02%2Fscanpix-20210102-170307-l.jpg&amp;scaleAfter=crop&amp;quality=30&amp;w=1008&amp;h=567_o40 1008w"/></p>

</html>
