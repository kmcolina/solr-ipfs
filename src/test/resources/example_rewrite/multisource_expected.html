<html>
 <head>
  <title>Some page</title>
  <link rel="stylesheet" type="text/css" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=1">
 </head>
 <body>
  <h1>srcset testing</h1>
  <p>An image with absolute srcset: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=2" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=3 1x, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=4 2x"></p>
  <p>An image with mixed casing and spacing: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=2" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=3 720w , http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=4 730W"></p>
  <p>An image with absolute srcset and post-space: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=2" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=3 1x, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=4 2x "></p>
  <p>An image with absolute srcset and substring trickery: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=2" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=2 1x, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=5 2x"></p>
  <p>An image with relative srcset: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=10" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=10 1x, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=11 2x"></p>
  <p>An image with relative srcset and substring trickery: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=21" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=21 1x, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=22 2x"></p>
  <p>An image with root srcset: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=30" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=30 1x, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=31 2x"></p>
  <p>Reproduction of specific problem: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=30" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=40 1008w"></p>
 </body>
</html>